otherlibs/unix/unixsupport_win32.c

```diff
@@ -55,7 +55,8 @@ value caml_win32_alloc_handle(HANDLE h)
     caml_alloc_custom(&handle_ops, sizeof(struct filedescr), 0, 1);
   Handle_val(res) = h;
   Descr_kind_val(res) = KIND_HANDLE;
-  CRT_fd_val(res) = NO_CRT_FD;
+  atomic_store_explicit(&CRT_fd_val(res), (intnat)NO_CRT_FD,
+                        memory_order_relaxed);
```

Comment:
(This applies to all the relaxed stores in this change)

What's the rationale behind relaxed atomics? I see at least here we're doing some non-atomic stores before which makes me wonder.